pkg/webhook/clusterresourceplacement/v1beta1/clusterresourceplacement_validating_webhook.go

```diff
@@ -0,0 +1,57 @@
+package v1alpha1
```

Comment:
how come this is v1alpha1?

Reply:
I think it's better to not adding another directory layer. In this case, it will be just another file. It's easier to remove the v1alpha1 later.